apps/web/playwright/fixtures/users.ts

```diff
@@ -229,12 +232,48 @@ export const createUsersFixture = (page: Page, workerInfo: WorkerInfo) => {
                 id: _user.id,
               },
             },
-            schedulingType: "COLLECTIVE",
-            title: "Team Event - 30min",
-            slug: "team-event-30min",
+            schedulingType: scenario.schedulingType ?? SchedulingType.COLLECTIVE,
+            title: teamEventTitle,
+            slug: teamEventSlug,
             length: 30,
           },
         });
+        if (scenario.teammates) {
+          // Create Teammate users
+          for await (const teammateObj of scenario.teammates) {
+            const teamUser = await prisma.user.create({
+              data: createUser(workerInfo, teammateObj),
+            });
+
+            // Add teammates to the team
+            await prisma.membership.create({
+              data: {
+                teamId: team.id,
+                userId: teamUser.id,
+                role: MembershipRole.MEMBER,
+                accepted: true,
+              },
+            });
+
+            // Add teammate to the host list of team event
+            await prisma.host.create({
+              data: {
+                userId: teamUser.id,
+                eventTypeId: teamEvent.id,
+                isFixed: scenario.schedulingType === SchedulingType.COLLECTIVE ? true : false,
+              },
+            });
+
+            const teammateFixture = createUserFixture(
+              await prisma.user.findUniqueOrThrow({
+                where: { id: teamUser.id },
+                include: userIncludes,
+              }),
+              store.page
+            );
+            store.users.push(teammateFixture);
+          }
+        }
```

Comment:
Add teammates in scenario to add them in the same team as the user. Also add them in the host list of the team event.
Add teammate to the userFixture for cleanup at the end of the test.